Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/SamplePaywallsList.swift

```diff
@@ -136,6 +138,12 @@ struct SamplePaywallsList: View {
                 } label: {
                     TemplateLabel(name: "Unrecognized paywall", icon: "exclamationmark.triangle")
                 }
+                
+                Button {
+                    self.display = .customerCenter
+                } label: {
+                    TemplateLabel(name: "Customer Center", icon: "person.fill.questionmark")
```

Comment:
I was actually adding a similar button in my [PR](https://github.com/RevenueCat/purchases-ios/pull/4053). I'm going to update the label to say `Customer Center (sheet)`, so it's clearer. Should we change the label here to `Customer Center (SwiftUI)` so it's more distinct? Maybe it's ok to leave this as is and just change it in other buttons.

Reply:
Ah good call! Will update the label as you suggested. Should we align on the icon as well?

Reply:
I feel we should create another section if we're going to have multiple buttons. As for the icon, we got a close one already :P. I think it's best for it to be slightly different... But honestly doesn't matter too much IMO.

Reply:
A section sounds good! I'll rebase this onto your branch and add both of our buttons to the section.